api/buildout.cfg

```diff
@@ -77,7 +88,7 @@ plone.rest = 2.0.0a1
 
 plone.volto =
 robotframework =
-robotframework-debuglibrary =
+robotframework-debuglibrary = <2.2
```

Comment:
Could you elaborate this? Did you find any problem?

Reply:
A version incompatibility pops up when adding the `[test]` extra requirements for those three mr.developer checkouts.  I tried to reproduce it to get you the specific details but now I'm getting a different version conflict.  I'm digging into that, but in the meantime I took the version of `robotframework-debuglibrary` that was conflicting and decremented it's minor version and that worked.

Reply:
See the commit messages of updated version pins from the `master` merge and subsequent commits I just pushed for more details.